src/AppBundle/Entity/Answer.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: lukas
+ * Date: 16.11.13
+ * Time: 22.30
+ */
+
+namespace AppBundle\Entity;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity()
+ * @ORM\Table(name="answers")
+ */
+class Answer
+{
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\GeneratedValue(strategy="AUTO")
+     */
+    private $id;
+    /**
+     * @ORM\Column(type="integer")
+     * @ORM\ManyToOne(targetEntity="questions")
+     */
+    private $questionId;
+    /**
+     * @ORM\Column(length=500)
+     */
+    private $text;
+    /**
+     * @ORM\Column(type="boolean")
+     */
+    private $correct;
+
```

Comment:
You should have oneToMany relation and a property here called questions. Which should map back to question entity, Look into comments on Answer class for documentation